leiden-journal-of-international-law.csl

```diff
@@ -0,0 +1,637 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" default-locale="en-GB" and="symbol" et-al-min="3" initialize-with="." sort-separator=" " initialize-with-hyphen="false" demote-non-dropping-particle="never">
+  <info>
+    <title>Leiden Journal of International Law</title>
+    <title-short>LJIL</title-short>
+    <id>"http://www.zotero.org/styles/leiden-journal-of-international-law.csl</id>
+    <link href="http://www.zotero.org/styles//leiden-journal-of-international-law.csl" rel="self"/>
+    <link href="https://www.cambridge.org/core/services/aop-file-manager/file/575adb7dfcaa6b122ec738b3" rel="documentation"/>
+    <author>
+      <name>Øyvind Ege</name>
+      <email>oyvind.ege@gmail.com</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="international law"/>
```

Comment:
You can only pick from a defined set of categories: http://docs.citationstyles.org/en/stable/specification.html#appendix-i-categories. In this case, use "law".